packages/ui-color-picker/src/ColorPicker/props.ts

```diff
@@ -199,6 +199,11 @@ type ColorPickerOwnProps = {
    */
   tooltip?: React.ReactNode
 
+  /**
+   * The id of the text input.
```

Comment:
I'd add ` One is generated if not supplied.`

Reply:
Added it, thank you.